specification/dnsresolver/resource-manager/Microsoft.Network/stable/2022-06-21/examples/DnsForwardingRuleset_Delete.json

```diff
@@ -0,0 +1,18 @@
+{
+  "parameters": {
+    "subscriptionId": "abdd4249-9f34-4cc6-8e42-c2e32110603e",
+    "resourceGroupName": "sampleResourceGroup",
+    "dnsForwardingRulesetName": "samplednsForwardingRulesetName",
+    "api-version": "2022-06-21"
```

Comment:
API version are typically the first of the month. Unsure if there's convention with respect to our launch date, but I'd otherwise suggest 2022-07-01.

Reply:
I will change the version after the ARM team compared the diff between the preview swagger and the new swagger.